packages/protocol/contracts/governance/Governance.sol

```diff
@@ -891,19 +867,22 @@ contract Governance is IGovernance, Ownable, Initializable, UsingBondedDeposits,
    */
   function isProposalPassing(uint256 proposalId) public view returns (bool) {
     Proposal storage proposal = proposals[proposalId];
-    // yesRatio can be undefined, which is not a problem for isGreaterThan
-    FractionUtil.Fraction memory yesRatio = FractionUtil.Fraction(
-      proposal.votes.yes,
-      proposal.votes.yes.add(proposal.votes.no)
+
+    uint256 yesNoVotes = proposal.votes.yes.add(proposal.votes.no);
+    if (yesNoVotes == 0) {
+      return false;
+    }
+    int256 yesRatio = toFixed(proposal.votes.yes).divide(
```

Comment:
This makes think. It's like we kinda say "we can sum up votes from 0 to 2^256-1" (uint256). But that's actually not true, the system will fail. So, in some sense, if a number ends being used with fixidity at some time, then it's a lie that we support that number range. We don't and we should make that apparent. And know that we have that limitation.

So votes, can go from 0 to `(2^(256-1)-1) - 10^24` ~ 10^76 [here](https://www.wolframalpha.com/input/?i=%282%5E%28256-1%29-1%29+-+10%5E24)